Makefile.defs

```diff
@@ -87,7 +87,8 @@ ifneq ($(V),0)
 endif
 endif
 
-GOLANGCILINT_WANT_VERSION = 1.51.2
+# renovate: datasource=docker depName=golangci/golangci-lint
+GOLANGCILINT_WANT_VERSION = v1.51.2
 GOLANGCILINT_IMAGE_SHA = sha256:be42f5df32011553183947c2b2502718bde485f566e15bbd2541f38a077e4d29
```

Comment:
Don't we need to update the sha as well? It's used in `make golangci-lint` when `golangci-lint` is not installed locally or its version doesn't match `$GOLANGCILINT_WANT_VERSION`.

Reply:
@tklauser i hope this gets automatically detected and updated via matching the `currentDigest` that i added to the matchstring

see https://github.com/mhofstetter/cilium/blob/04ccfde61cb9a6b855dcc437a049e4cf43e1c431/.github/renovate.json5#L275

Reply:
:sparkles: nice, thanks for the explanation.

Reply:
we'll see whether it works as intended once the PR is here :D 

@tklauser @aanm btw: i guess we're using the Renovate App from the GitHub Marketplace and not the standalone renovate runner? what's our configured default renovate schedule?

Reply:
@mhofstetter we are using the "Renovate App from the GitHub Marketplace". It depends on the package, some packages are defined to run on Friday, for example [here](https://github.com/cilium/cilium/blob/master/.github/renovate.json5#L69)